test/metabase/query_processor/middleware/process_userland_query_test.clj

```diff
@@ -124,35 +124,32 @@
 (deftest record-view-log-when-process-userland-query-test
   (testing "record a view log with only card id"
     (mt/with-temp [:model/Card card {:dataset_query (mt/mbql-query users)}]
-      (mt/with-model-cleanup [:model/ViewLog]
-        (qp/process-userland-query (assoc
-                                    (:dataset_query card)
-                                    :info {:card-id (:id card)}))
+      (qp/process-userland-query (assoc
+                                  (:dataset_query card)
+                                  :info {:card-id (:id card)}))
 
-        (is (true? (t2/exists? :model/ViewLog :model "card" :model_id (:id card) :user_id nil))))))
+      (is (true? (t2/exists? :model/ViewLog :model "card" :model_id (:id card) :user_id nil)))))
 
   (testing "record a view log with card id and executed by"
     (mt/with-temp [:model/Card card {:dataset_query (mt/mbql-query users)}]
-      (mt/with-model-cleanup [:model/ViewLog]
-        (qp/process-userland-query (assoc
-                                    (:dataset_query card)
-                                    :info {:card-id (:id card)
-                                           :executed-by (mt/user->id :rasta)}))
+      (qp/process-userland-query (assoc
+                                  (:dataset_query card)
+                                  :info {:card-id (:id card)
+                                         :executed-by (mt/user->id :rasta)}))
 
-        (is (true? (t2/exists? :model/ViewLog :model "card" :model_id (:id card) :user_id (mt/user->id :rasta)))))))
+      (is (true? (t2/exists? :model/ViewLog :model "card" :model_id (:id card) :user_id (mt/user->id :rasta))))))
 
   (testing "skip if context is dashboard or collection"
     (mt/with-temp [:model/Card card {:dataset_query (mt/mbql-query users)}]
-      (mt/with-model-cleanup [:model/ViewLog]
```

Comment:
Out of curiosity, why remove the `with-model-cleanup` here?

Reply:
I think in general, using `with-model-cleanup` for AuditLog and ViewLog doesn't make too much sense. Lots of other tests are writing to these tables implicitly by triggering events, so the tables will be filled with "fake" data anyway. So cleaning up these models just adds work to the tests without any benefit.